content/release-10-17-0/platform-10-17-0.md

```diff
@@ -155,6 +166,22 @@ Build version</th>
 </tr>
 </thead><tbody>
 
+<tr>
+<td>Administration</td>
+<td>Fixed an issue where users with inventory roles only could not add new groups.</td>

```

Comment:
```suggestion
<td>Fixed an issue where users with only inventory roles could not add new groups.</td>

```

Reply:
I would rather change it to "where users that only have inventory roles". Both other sentences sound strange :-). Is "that" ok here? I mean the users are not really persons.

Reply:
i suggest "which" which is more commonly used in written language (and less informal)